WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -326,6 +328,18 @@ class MySiteViewModel @Inject constructor(
         return orderForDisplay(infoItem, cardsResult, dynamicCards, siteItems)
     }
 
+    private fun onTodaysStatsCardFooterLinkClick() {
+        val selectedSite = requireNotNull(selectedSiteRepository.getSelectedSite())
+        // Todo: @ajeshrpai add analytics here
+        _onNavigation.value = Event(SiteNavigationAction.OpenTodaysStats(selectedSite))
+    }
+
+    private fun onTodaysStatsCardClick() {
+        val selectedSite = requireNotNull(selectedSiteRepository.getSelectedSite())
+        // Todo@ajeshrpai add analytics here
+        _onNavigation.value = Event(SiteNavigationAction.OpenTodaysStats(selectedSite))
+    }
+
```

Comment:
Since both footer link click and card click should navigate to `Stats -> Days` tab, I wonder if we can extract logic into a common method. Wdyt?

Reply:
@ashiagr Yeah, I guess we can extract the logic into a common method. 👍 . Done in f1c7cd4